components/MaskedTransition/examples/MaskedTransitionTypicalUse.swift

```diff
@@ -174,8 +184,48 @@ private class ModalViewController: UIViewController {
     label.numberOfLines = 0
     label.lineBreakMode = .byWordWrapping
     label.text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In aliquam dolor eget orci condimentum, eu blandit metus dictum. Suspendisse vitae metus pellentesque, sagittis massa vel, sodales velit. Aliquam placerat nibh et posuere interdum. Etiam fermentum purus vel turpis lobortis auctor. Curabitur auctor maximus purus, ac iaculis mi. In ac hendrerit sapien, eget porttitor risus. Integer placerat cursus viverra. Proin mollis nulla vitae nisi posuere, eu rutrum mauris condimentum. Nullam in faucibus nulla, non tincidunt lectus. Maecenas mollis massa purus, in viverra elit molestie eu. Nunc volutpat magna eget mi vestibulum pharetra. Suspendisse nulla ligula, laoreet non ante quis, vehicula facilisis libero. Morbi faucibus, sapien a convallis sodales, leo quam scelerisque leo, ut tincidunt diam velit laoreet nulla. Proin at quam vel nibh varius ultrices porta id diam. Pellentesque pretium consequat neque volutpat tristique. Sed placerat a purus ut molestie. Nullam laoreet venenatis urna non pulvinar. Proin a vestibulum nulla, eu placerat est. Morbi molestie aliquam justo, ut aliquet neque tristique consectetur. In hac habitasse platea dictumst. Fusce vehicula justo in euismod elementum. Ut vel malesuada est. Aliquam mattis, ex vel viverra eleifend, mauris nibh faucibus nibh, in fringilla sem purus vitae elit. Donec sed dapibus orci, ut vulputate sapien. Integer eu magna efficitur est pellentesque tempor. Sed ac imperdiet ex. Maecenas congue quis lacus vel dictum. Phasellus dictum mi at sollicitudin euismod. Mauris laoreet, eros vitae euismod commodo, libero ligula pretium massa, in scelerisque eros dui eu metus. Fusce elementum mauris velit, eu tempor nulla congue ut. In at tellus id quam feugiat semper eget ut felis. Nulla quis varius quam. Nullam tincidunt laoreet risus, ut aliquet nisl gravida id. Nulla iaculis mauris velit, vitae feugiat nunc scelerisque ac. Vivamus eget ligula porta, pulvinar ex vitae, sollicitudin erat. Maecenas semper ornare suscipit. Ut et neque condimentum lectus pulvinar maximus in sit amet odio. Aliquam congue purus erat, eu rutrum risus placerat a."
-    label.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+    label.translatesAutoresizingMaskIntoConstraints = false
     view.addSubview(label)
+
+    var leftOffset: CGFloat = 0
+    var rightOffset: CGFloat = 0
+    var topOffset: CGFloat = 0
+    var bottomOffset: CGFloat = 0
+    if #available(iOS 11.0, *) {
+      leftOffset += additionalSafeAreaInsets.left
+      rightOffset += additionalSafeAreaInsets.right
+      topOffset += additionalSafeAreaInsets.top
+      bottomOffset += additionalSafeAreaInsets.bottom
+    }
+
+    NSLayoutConstraint(item: view,
```

Comment:
_Tip:_ Some of this layout logic could be extracted into another method

Reply:
Agreed and done!